ui/src/containers/CreateVolume.js

```diff
@@ -54,6 +55,8 @@ const ActionContainer = styled.div`
 const CreateVolumeLayout = styled.div`
   display: inline-block;
   margin-top: ${padding.base};
+  overflow-y: scroll;
```

Comment:
Is it intended to display scrollbars by default even for non scrollable content ? You might prefer to use `auto` instead